e2e/tests/addHearingBookingDetails_test.js

```diff
@@ -0,0 +1,87 @@
+Feature('Add hearing booking details');
+
+Before(async (I, caseViewPage, submitApplicationEventPage) => {
+  I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+  caseId = await I.grabTextFrom('.heading-h1');
+  caseViewPage.goToNewActions(config.applicationActions.submitCase);
+  submitApplicationEventPage.giveConsent();
+  I.continueAndSubmit();
+  I.signOut();
+});
+
+Scenario('Enter hearing details and submitting as HMCTS admin', async (I, caseViewPage, loginPage, addHearingBookingDetailsEventPage) => {
+  loginPage.signIn(config.hmctsAdminEmail, config.hmctsAdminPassword);
+  I.navigateToCaseDetails(caseId);
+  I.see(caseId);
+  caseViewPage.goToNewActions(config.administrationActions.addHearingDetails);
+  addHearingBookingDetailsEventPage.addHearing();
+  await addHearingBookingDetailsEventPage.enterHearingDetails(hearingDetails[0]);
+  addHearingBookingDetailsEventPage.addHearing();
+  await addHearingBookingDetailsEventPage.enterHearingDetails(hearingDetails[1]);
+  I.continueAndProvideSummary('summary', 'description');
+  I.seeEventSubmissionConfirmation(config.administrationActions.addHearingDetails);
+  caseViewPage.selectTab(caseViewPage.tabs.hearings);
+  seeAnswersInHearingTab(I);
+});
+
+Scenario('Enter hearing details and submitting as Judiciary', async (I, caseViewPage, loginPage, addHearingBookingDetailsEventPage) => {
+  loginPage.signIn(config.judiciaryEmail, config.judiciaryPassword);
+  I.navigateToCaseDetails(caseId);
+  I.see(caseId);
+  caseViewPage.goToNewActions(config.administrationActions.addHearingDetails);
+  addHearingBookingDetailsEventPage.addHearing();
+  await addHearingBookingDetailsEventPage.enterHearingDetails(hearingDetails[0]);
+  addHearingBookingDetailsEventPage.addHearing();
+  await addHearingBookingDetailsEventPage.enterHearingDetails(hearingDetails[1]);
+  I.continueAndProvideSummary('summary', 'description');
+  I.seeEventSubmissionConfirmation(config.administrationActions.addHearingDetails);
+  caseViewPage.selectTab(caseViewPage.tabs.hearings);
+  seeAnswersInHearingTab(I);
+});
+
+
+Scenario('Enter hearing details and submitting as Gatekeeper', async (I, caseViewPage, loginPage, addHearingBookingDetailsEventPage) => {
+  loginPage.signIn(config.gateKeeperEmail, config.gateKeeperPassword);
+  I.navigateToCaseDetails(caseId);
+  I.see(caseId);
+  caseViewPage.goToNewActions(config.administrationActions.addHearingDetails);
+  addHearingBookingDetailsEventPage.addHearing();
+  await addHearingBookingDetailsEventPage.enterHearingDetails(hearingDetails[0]);
+  addHearingBookingDetailsEventPage.addHearing();
+  await addHearingBookingDetailsEventPage.enterHearingDetails(hearingDetails[1]);
+  I.continueAndProvideSummary('summary', 'description');
+  I.seeEventSubmissionConfirmation(config.administrationActions.addHearingDetails);
+  caseViewPage.selectTab(caseViewPage.tabs.hearings);
+  seeAnswersInHearingTab(I);
+});
+
+function seeAnswersInHearingTab(I) {
```

Comment:
Is this needed?

Reply:
Rather than repeating it three times I thought it was easier to have one function to reuse